Content.Client/_DV/Curation/UI/Cwoink/CwoinkControl.xaml.cs

```diff
@@ -0,0 +1,282 @@
+using Content.Client.Administration.UI.CustomControls;
+using Content.Client.UserInterface.Controls;
+using Content.Shared.Administration;
+using Content.Shared.CCVar;
+using Robust.Client.AutoGenerated;
+using Robust.Client.Console;
+using Robust.Client.UserInterface;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.XAML;
+using Robust.Shared.Configuration;
+using Robust.Shared.Network;
+using Robust.Shared.Utility;
+
+namespace Content.Client._DV.Curation.UI.Cwoink;
+
+/// <summary>
+/// This window connects to a CwoinkSystem channel. CwoinkSystem manages the rest.
+/// </summary>
+[GenerateTypedNameReferences]
+public sealed partial class CwoinkControl : Control
+{
+    [Dependency] private readonly IClientAdminManager _adminManager = default!;
+    [Dependency] private readonly IClientConsoleHost _console = default!;
+    [Dependency] private readonly IUserInterfaceManager _ui = default!;
+    [Dependency] private readonly IConfigurationManager _cfg = default!;
+    public CuratorCHelpUIHandler CHelpHelper = default!;
+
+    private PlayerInfo? _currentPlayer;
+
+    public CwoinkControl()
+    {
+        RobustXamlLoader.Load(this);
+        IoCManager.InjectDependencies(this);
+
+        var newPlayerThreshold = 0;
+        _cfg.OnValueChanged(CCVars.NewPlayerThreshold, (val) => { newPlayerThreshold = val; }, true);
+
+        var uiController = _ui.GetUIController<CHelpUIController>();
+        if (uiController.UIHelper is not CuratorCHelpUIHandler helper)
+            return;
+
+        CHelpHelper = helper;
+
+        _adminManager.AdminStatusUpdated += UpdateButtons;
+        UpdateButtons();
+
+        CuratorOnly.OnToggled += args => PlaySound.Disabled = args.Pressed;
+
+        ChannelSelector.OnSelectionChanged += sel =>
+        {
+            _currentPlayer = sel;
+            SwitchToChannel(sel?.SessionId);
+            ChannelSelector.PlayerListContainer.DirtyList();
+        };
+
+        ChannelSelector.OverrideText += (info, text) =>
+        {
+            var sb = new StringBuilder();
+
+            if (info.Connected)
+                sb.Append(info.ActiveThisRound ? '⚫' : '◐');
+            else
+                sb.Append(info.ActiveThisRound ? '⭘' : '·');
+
+            sb.Append(' ');
+            if (CHelpHelper.TryGetChannel(info.SessionId, out var panel) && panel.Unread > 0)
+            {
+                if (panel.Unread < 11)
+                    sb.Append(new Rune('➀' + (panel.Unread-1)));
+                else
+                    sb.Append(new Rune(0x2639)); // ☹
+                sb.Append(' ');
+            }
+
+            // Mark antagonists with symbol
+            if (info.Antag && info.ActiveThisRound)
+                sb.Append(new Rune(0x1F5E1)); // 🗡
+
+            // Mark new players with symbol
+            if (IsNewPlayer(info))
+                sb.Append(new Rune(0x23F2)); // ⏲
+
+            sb.AppendFormat((string)"\"{0}\"", (object?)text);
+
```

Comment:
Just split these up, no need to use format just to add two strings together
```suggestion

            sb.Append('"');
            sb.Append(text);
            sb.Append('"');

```